src/theme/components.js

```diff
@@ -18,6 +18,9 @@ const components = {
     },
     styleOverrides: {
       root: {
+        '&.btn-all': {
+          padding: '4px 16px',
+        },
```

Comment:
If this should be added to all buttons, better add it directly to `root` without this extra class.
See how I did it in https://github.com/mozilla/perfcompare/pull/666